pkg/model/gcemodel/context.go

```diff
@@ -37,7 +37,7 @@ func (c *GCEModelContext) LinkToNetwork() *gcetasks.Network {
 func (c *GCEModelContext) NameForNetwork() string {
 	networkName := c.Cluster.Spec.NetworkID
 	if networkName == "" {
-		networkName = "default"
+		networkName = gce.SafeClusterName(c.Cluster.ObjectMeta.Name)
```

Comment:
This is because there's a pretty low limit on the number of ipalias ranges per network (I think it's 5?) , and so you want to have a network for each cluster?  I think this makes sense, but if so a comment would be great!